specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LongTermRetentionPolicies.json

```diff
@@ -243,6 +220,33 @@
           "readOnly": true
         }
       }
+    },
+    "LongTermRetentionPolicyProperties": {
+      "description": "Properties of a long term retention policy",
+      "type": "object",
+      "properties": {
+        "makeBackupsImmutable": {
+          "description": "The setting whether to make LTR backups immutable",
+          "type": "boolean"
+        },
+        "weeklyRetention": {
+          "description": "The weekly retention policy for an LTR backup in an ISO 8601 format.",
+          "type": "string"
+        },
+        "monthlyRetention": {
+          "description": "The monthly retention policy for an LTR backup in an ISO 8601 format.",
+          "type": "string"
+        },
+        "yearlyRetention": {
+          "description": "The yearly retention policy for an LTR backup in an ISO 8601 format.",
+          "type": "string"
+        },
+        "weekOfYear": {
+          "format": "int32",
+          "description": "The week of year to take the yearly backup in an ISO 8601 format.",
```

Comment:
>in an ISO 8601 format [](http://example.com/codeflow?start=69&length=21)

format is int32 ?

Reply:
This is not actually in an ISO 8601 format, I can look into fixing this in a later PR but I would like to unblock this PR before the deadline on Friday